.github/workflows/build-test.yml

```diff
@@ -729,20 +766,20 @@ jobs:
           VSC_JUPYTER_FORCE_LOGGING: 1
           VSC_PYTHON_FORCE_LOGGING: 1
           VSC_JUPYTER_CI_RUN_NON_PYTHON_NB_TEST: 1
-          VSC_JUPYTER_REMOTE_NATIVE_TEST: ${{ contains(matrix.test-suite, 'remote') }}
-          VSC_JUPYTER_NON_RAW_NATIVE_TEST: ${{ contains(matrix.test-suite, 'nonRawNotebook') }}
-          VSC_JUPYTER_CI_RUN_JAVA_NB_TEST: ${{ contains(matrix.test-suite, 'conda') }}
-          VSC_JUPYTER_CI_IS_CONDA: ${{ contains(matrix.test-suite, 'conda') }}
+          VSC_JUPYTER_REMOTE_NATIVE_TEST: ${{ matrix.jupyter == 'remote' }}
```

Comment:
Should probably clean this up as well, and have just one single test step, instead of the 3 that we have. however I believe what we have now with the current change, its much simpler to add new permutations & later we can clean this.

Plan was to work on Windows tests, but had to move to other issues.